src/app/components/Header.tsx

```diff
@@ -33,16 +32,20 @@ export default async function Header() {
         <MenuItem key="home" component={NextLink} href="/">
           Home
         </MenuItem>
-        <MenuItem
-          key="patient-summary"
-          component={NextLink}
-          href="/patient-summary"
-        >
-          Patient Summary View
-        </MenuItem>
-        <MenuItem key="dashboard" component={NextLink} href="/shared-links">
-          Dashboard
-        </MenuItem>
+        {isAuthenticated && (
+          <MenuItem
+            key="patient-summary"
+            component={NextLink}
+            href="/patient-summary"
+          >
+            Patient Summary View
+          </MenuItem>
+        )}
+        {isAuthenticated && (
+          <MenuItem key="dashboard" component={NextLink} href="/shared-links">
+            Dashboard
+          </MenuItem>
```

Comment:
Hello @yassinedorbozgithub, I suggest we group these two renders under one condition. What do you think?
```suggestion
    {isAuthenticated && (
          <>
            <MenuItem
              key="patient-summary"
              component={NextLink}
              href="/patient-summary"
            >
              Patient Summary View
            </MenuItem>

            <MenuItem key="dashboard" component={NextLink} href="/shared-links">
              Dashboard
            </MenuItem>
          </>
        )}
```

Reply:
Hi @amalessid, comment addressed :+1: